src/backend/columnar/sql/udfs/alter_columnar_table_reset/10.0-1.sql

```diff
@@ -5,8 +5,39 @@ CREATE OR REPLACE FUNCTION pg_catalog.alter_columnar_table_reset(
     compression bool DEFAULT false,
     compression_level bool DEFAULT false)
     RETURNS void
-    LANGUAGE C
-AS 'MODULE_PATHNAME', 'alter_columnar_table_reset';
+    LANGUAGE plpgsql AS
```

Comment:
As a policy, we generally do not change past SQL files, since there's a high risk of diverging schemas between upgrades vs. new installs. Instead we always write a new version (11.1-1.sql) and replay all scripts for new installs. In many cases this also makes it easier to write a correct downgrade script, because you can simply #include the previous version from the downgrade script.

Eventually, once we have phased out support for an older PostgreSQL version, we can collapse all SQL scripts up to the lowest version that supports one of the currently supported PostgreSQL versions.

For instance, Citus 9.5 is the first version that supported PostgreSQL 13, and PostgreSQL 13 is the lowest version supported by Citus 11.0. Hence, we can collapse all scripts from 8.0-1 up to 9.5-1 into a single script, because there is no scenario in which a user could upgrade directly from 9.4 or older to 11.0. When we collapse, we can also remove redundant creations.

Reply:
Done, thank you.